src/router/index.js

```diff
@@ -16,11 +16,10 @@ const routes = [
     component: Applications
   },
   {
-    path: '/apps/edit/:id?',
+    path: '/edit/:appId/:appObjectId?',
```

Comment:
O prefixo `/apps` tem que estar em todas as rotas porque o admin naturalmente já possui outras rotas...

```suggestion
    path: '/apps/edit/:appId/:appObjectId?',
```